ci-operator/step-registry/gcp/provision/private-dns-zone/gcp-provision-private-dns-zone-commands.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
How about splitting the script into two, one is for creating private zones in host project, another one is for creating private zones in service project. So that we can easily opt-in any step in workflow if needed, save a lot of `if` conditions.